### ISSUE TITLE
Small typos in some howtos

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3883]
|<img alt="avu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| avu reported|
|-|-|
|Trac ID|trac#3883|
|Type|defect|
|Created|2009-06-18 01:46:29Z|

I found some small typos while reading the supplied howtos, so I made a patch.

Attachments:

* [twisted-docs.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/e8c/e8c1bea84ec9b7eb4accf9eb102de4ce16d74eb9/dc7f88dba9df86d19190ecf603176587b89ab399.patch) (2662 bytes) - added by avu on 2009-06-18 01:46:49Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3883 3883
type__defect defect
reporter__avu avu
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1245289589000000 1245289589000000
changetime__1245311575000000 1245311575000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_1"></a>|@glyph set owner to avu|
|-|-|

Did you intend to put this up for review?

**Comment by twisted-trac** [comment]
|<img alt="avu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|avu commented|
|-|-|

sigh.

**Comment by twisted-trac** [comment]
|<img alt="avu's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|avu removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)<a name="note_4"></a>|@mwhudson set owner to @mwhudson|
|-|-|

Looks fine, I'll land it.

**Comment by twisted-trac** [comment]
|[<img alt="mwhudson's avatar" src="https://avatars.githubusercontent.com/u/672022?s=50" width="50" height="50">](https://github.com/mwhudson)<a name="note_5"></a>|@mwhudson set status to `closed`|
|-|-|

(In [27020]) Merge twisted-docs.patch: Fix some typos in the howtos.

Author: avu
Reviewer: mwh 
Fixes: [#3883](https://github.com/twisted/twisted/issues/3883)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
